migrations/V115__DROP_GRANTS_FROM_PP_NON_CLAIM_BASED_PAYMENT_TYPE.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TYPE PP_NON_CLAIM_BASED_PAYMENT_TYPE RENAME TO PP_NON_CLAIM_BASED_PAYMENT_TYPE_OLD;
+
+CREATE TYPE PP_NON_CLAIM_BASED_PAYMENT_TYPE AS ENUM (
+  'ADVANCED_PAYMENT',
+  'BUNDLED_EPISODE_OF_CARE',
+  'CAPITATION_POPULATION_BASED_FULL',
+  'CAPITATION_POPULATION_BASED_PARTIAL',
+  'CARE_COORDINATION_MANAGEMENT_FEE',
+  'GLOBAL_BUDGET',
+  'INCENTIVE_PAYMENT',
+  'MAPD_SHARED_SAVINGS',
+  'SHARED_SAVINGS',
+  'OTHER'
+  );
+
+ALTER TABLE plan_payments

```

Comment:
@OddTomBrooks, I think that this migration should handle in case the data has `GRANTS` selected. Even though the data was audited in prod and shouldn't have it selected, the migration should remove that selection if selected. (for example if it is selected anywhere in test or impl, this migration will fail currently.